facebook.js

```diff
@@ -114,12 +115,12 @@ const show_facebook_cv_tags = function() {
 
 const observer = new MutationObserver(function(mutations) {
     mutations.forEach(function(mutation) {
-        show_facebook_cv_tags();
+        setTimeout(function(){show_facebook_cv_tags(mutation.target);},1500);
```

Comment:
waiting a specific amount of time is never a good idea, as it will depend on the persons computer. The only good fit of a setTimeout on JS execution engines are _.defer or setTimeout(fn, 0) that allow to bypass the direct execution and wait for rendering (in most cases, yeah, js is bugged as hell)